### PR TITLE
Gig Economy+, Matt PWA Post, Tues 26

### PR DESCRIPTION
**Pull request by bcardarella**

### CODE REVIEW

**Comment by sbatson5** [comment]
The file name here is missing the `2` in front of year

File: source/posts/2017-09-26-gig-economy-and-developing-nations-and-pwas.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+---
+layout: post
+title: "The gig economy and developing nations; how Progressive Web Apps will help save the world."
+social: true
+author: Matt Ludwig
+twitter: "@migdesigner"
+github:
+summary: “With the gig economy services expanding to developing regions more and more every quarter, the need for a new wave of exceptional offline user experiences is just a Progressive Web App away."
```
Comment (brzpegasus) [inline source/posts/2017-09-26-gig-economy-and-developing-nations-and-pwas.md]: The summary starts with a smart quotes. This should be replaced with regular double quotes.

File: source/posts/2017-09-26-gig-economy-and-developing-nations-and-pwas.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+Don’t get me wrong, I too feel the struggle is real when my 60mb WiFi goes down and I have to [TOTO](http://starecat.com/content/wp-content/uploads/when-my-family-praises-me-for-fixing-the-wifi-and-all-i-did-was-restart-the-router-hackerman-kung-fury.jpg). But that’s far from a ‘struggle’ in many daily experiences of a huge growing percentage of gig workers across developing nations. Many companies (state-run and private) can’t afford to invest in native mobile solutions as the amount of work required to maintain these complex stacks for synonymous on/offline performance is just too costly for native apps. All too often ignored is the fact that all of this is overwhelmingly possible to solve with a Progressive Web App strategy with no workarounds or hacks. [Even for just this reason alone](http://www.gartner.com/newsroom/id/3415117).The struggle is really real, but it doesn’t need to be.
```
Comment (bcardarella) [inline source/posts/2017-09-26-gig-economy-and-developing-nations-and-pwas.md]: TOTO: This is a link to an image. Was this meant to be embedded in the post?

Also, any images should be imported into imgur to ensure we don't get flagged for hotlinking

File: source/posts/2017-09-26-gig-economy-and-developing-nations-and-pwas.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+## “But our corporate WiFi is never down!”
+Liar. I’ve worked in, visited, and designed software solutions in countless major corporate offices across the globe, and the common trait among them all are glitchy internet connections. All of them. It’s not for a lack of preventative maintenance, availability, or skill; corporate WLAN’s are just fickle beasts. Fickle, in that they suffer from ISP blips/outages (that’s just life), and also because their inherent designs for access and usage can be something out of a [David Bowie movie](https://jaysanalysis.files.wordpress.com/2015/04/labyrinth_movie.jpg): all kinds of hoops to jump through to get on the network, then firewall issues with CorpA device on CorpB’s network, and firewalls blocking certain “untrusted” (or more likely, “unknown”) URLs, etc..
```
Comment (bcardarella) [inline source/posts/2017-09-26-gig-economy-and-developing-nations-and-pwas.md]: Another image that is hotlinked. Should it be embedded?

**Pull request merged**